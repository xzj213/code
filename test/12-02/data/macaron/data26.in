18
0 1321 0 1321 0 1321 0 1321 1321 0 0 1321 0 1321 0 0 0 1321
1321 0 0 1321 1321 1321 0 1321 0 0 1321 1321 1321 1321 0 1321 0 1321
0 0 1321 0 1321 1321 0 1321 1321 1321 0 1321 1321 1321 1321 1321 0 1321
0 0 0 1321 1321 1321 1321 1321 0 1321 0 0 0 1321 0 0 1321 0
1321 0 0 0 1321 0 1321 1321 0 1321 1321 0 0 1321 0 0 1321 1321
1321 0 0 1321 1321 1321 0 1321 1321 1321 1321 1321 0 0 0 0 1321 1321
0 1321 1321 1321 1321 1321 1321 0 0 1321 0 0 1321 1321 0 1321 1321 1321
0 0 1321 0 0 1321 1321 0 0 0 1321 1321 1321 1321 1321 0 1321 1321
0 1321 1321 1321 0 1321 0 1321 1321 1321 1321 1321 1321 0 1321 0 0 1321
0 0 0 1321 1321 1321 0 1321 1321 0 0 0 0 1321 0 0 0 0
0 0 0 0 1321 0 1321 1321 1321 0 0 1321 0 1321 0 1321 1321 1321
1321 1321 0 1321 1321 1321 1321 0 1321 0 1321 1321 0 1321 1321 1321 0 1321
1321 1321 0 1321 0 0 1321 1321 1321 0 0 1321 0 1321 0 0 1321 1321
1321 0 0 1321 1321 1321 0 0 1321 0 1321 0 0 0 1321 1321 1321 1321
0 0 0 1321 1321 1321 1321 1924 1321 1321 0 1321 1321 1321 1321 1321 0 0
0 1321 1321 0 1321 1321 1321 1321 1321 0 1321 0 1321 1321 0 0 1321 1321
1321 0 0 1321 0 0 0 0 1321 1321 0 1321 1321 1321 1321 1321 0 0
1321 1321 0 1321 0 0 1321 0 1321 1321 1321 1321 1321 0 0 0 1321 1321
