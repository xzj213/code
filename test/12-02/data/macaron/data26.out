0 685 1321 1924 
